src/main/scala/uk/gov/hmrc/nonrep/attachment/server/ServiceConfig.scala

```
@@ -33,11 +33,20 @@ class ServiceConfig(val servicePort: Int = 8000) {
   val signaturesServicePort: Int = signaturesServiceUri.getPort
   val signingProfile: String = signaturesParams.getString("signing-profile")
 
+  private val systemParams = config.getObject(s"$appName.system-params").toConfig
+  val maxBufferSize: Int = systemParams.getInt("maxBufferSize")
+  val maxBatchSize: Int = systemParams.getInt("maxBatchSize")
+  val closeOnEmptyReceive: Boolean = systemParams.getBoolean("closeOnEmptyReceive")
+  val waitTimeSeconds: Int = systemParams.getInt("waitTimeSeconds")
+
+  val queueUrl: String = sys.env.getOrElse("ATTACHMENT_SQS", s"https://sqs.eu-west-2.amazonaws.com/205520045207/$env-nonrep-attachment-queue")
```

Perhaps it would be better to throw an exception in a deployed environment if the system property is not set? This is consistent with other system properties, and uou can allow it to work locally without the property, e.g. see `glacierNotificationsSnsTopicArn`.